trusted_os/rpc.go

```diff
@@ -308,3 +308,28 @@ func (r *RPC) Reboot(_ *any, _ *bool) error {
 
 	return nil
 }
+
+// ConsoleLog returns the console log for the current running OS & applet.
+func (r *RPC) ConsoleLog(_ *any, ret *[]byte) error {
+	if ret == nil {
+		return errors.New("nil buffer passed")
+	}
+
+	*ret = getConsoleLogs()
```

Comment:
This will only return something in debug mode, as opposed as the other function. Maybe this is something you want to mention here or not? Not blocking for this PR!

Reply:
Yup - that's expected, thanks for calling out though!